lib/vagrant-service-manager/services/open_shift.rb

```diff
@@ -6,12 +6,14 @@ class OpenShift < ServiceBase
       def initialize(machine, env)
         super(machine, env)
         @service_name = 'openshift'
-        @extra_cmd = build_extra_command
       end
 
       def execute
         if service_start_allowed?
-          command = "#{@extra_cmd} sccli openshift"
+          command = 'sccli openshift'
```

Comment:
> This underscore help to dynamically build the OpenShift class name. Otherwise it would difficult to do so with 'openshift' where there is no separation.

What do you mean? Are you saying that the file name as a bearing on how some code behaves? Then I give an additional vote to renaming to _openshift.rb_. Relying on the filename is wrong.